### PR TITLE
Filter channel id

### PR DESCRIPTION
**Pull request by dongrie**
Ethereum chain module's event query is modified to return only events from a specific channel

### CODE REVIEW

File: pkg/relay/ethereum/events.go [hunk 1]
```diff
@@ -160,6 +163,9 @@ func (chain *Chain) findRecvPacketEvents(ctx core.QueryContext, fromHeight uint6
 		if err != nil {
 			return nil, fmt.Errorf("failed to parse RecvPacket event: err=%v, log=%v", err, log)
 		}
+		if event.Packet.SourceChannel != chain.Path().ChannelID {
```
Comment (siburu) [inline pkg/relay/ethereum/events.go:166]: In `findWriteAckEvents` you chose `DestinationChannel`, but you chose `SourceChannel` here. Why?

File: pkg/relay/ethereum/events.go [hunk 2]
```diff
@@ -81,6 +81,9 @@ func (chain *Chain) findSentPackets(ctx core.QueryContext, fromHeight uint64) (c
 		if err != nil {
 			return nil, fmt.Errorf("failed to parse SendPacket event: err=%v, log=%v", err, log)
 		}
+		if sendPacket.SourceChannel != chain.Path().ChannelID {
+			continue
+		}
```
Comment (siburu) [inline pkg/relay/ethereum/events.go:86]: As a result of the discussion with @bluele, it was decided that port id should also be compared.

**Review by siburu — COMMENTED** [review]
@dongrie I have put two comments. Please check them.

File: pkg/relay/ethereum/events.go (hunk 1 above)
Comment (dongrie) [reply pkg/relay/ethereum/events.go]: Fix to `DestinationChannel`

File: pkg/relay/ethereum/events.go (hunk 2 above)
Comment (dongrie) [reply pkg/relay/ethereum/events.go:86]: Added check `portID`

Comment (siburu) [reply pkg/relay/ethereum/events.go:86]: @dongrie replace `&&` with `||` ...

Comment (dongrie) [reply pkg/relay/ethereum/events.go:86]: orz.... sorry, Fixed
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/38/commits/6eafa2e553de62ee44de1b2a460ee3e1f547b6b6

**Review by siburu — APPROVED** [review]
@dongrie LGTM! Thank you :bow:

**Pull request merged**